composables/useFindUser.ts

```diff
@@ -0,0 +1,13 @@
+export const useFindUser = async (username: string) => {
```

Comment:
- Follow up question regarding caching: Would it still be meaningful to have caching setup if I end up not using useUserIdByUsername in the validate portion of `definePageMeta`?
- In other words, even if I don't use this composable in the `validate` function (then technically the page wouldn't be making two repetitive requests via useUserIdByUsername - once in validate, once in script setup), would the caching of the userId be helpful later on if a user who visited /boyeonihn/0228 later visits another page with the same username (e.g. /boyeonihn/1123)?

Reply:
I tried implementing the caching in `useUserIdByUsername` -- everything works fine up until the 'params' part -- I got this error saying `nuxt` does not exist. 

![CleanShot 2024-05-01 at 10 09 46](https://github.com/boyeonihn/manyaday/assets/95451902/3d78be4d-fca5-4f7c-915f-40eebd6a3e58)


If I comment out params, the rest of the useAsyncData works fine.

Reply:
@eunjae-lee 
Another follow-up question 😂 (sorry):

I was looking at the useAsyncData params section that you gave me the link for [here](https://nuxt.com/docs/api/composables/use-async-data#params):

![CleanShot 2024-05-01 at 11 12 04](https://github.com/boyeonihn/manyaday/assets/95451902/60c79d2c-7853-42be-b0be-24b2e922ab33)
Since the getCachedData function we passed (in your written example above) is identical to this default function, should we even have to pass a function in `getCachedData` in our `options` for useAsyncData? 


And to utilize caching data, the doc above mentioned I have to have `payloadExtraction` enabled, which requires setting config like [this](https://nuxt.com/docs/guide/going-further/experimental-features):
![CleanShot 2024-05-01 at 11 15 48](https://github.com/boyeonihn/manyaday/assets/95451902/526ed6b7-bbe2-44b6-b998-6d9737998464)

Did I read that correctly?